server/chat-plugins/mafia.ts

```diff
@@ -1588,6 +1600,26 @@ class Mafia extends Rooms.RoomGame<MafiaPlayer> {
 		if (!setting) this.clearVotes('novote');
 		this.updatePlayers();
 	}
+	setVotelock(user: User, setting: boolean) {
```

Comment:
Do these (this and `setVoting`) necessarily need to be new methods? They don't seem to be used anywhere else, so putting them in the commands would probably work fine.

Reply:
for what it's worth setSelfVote and setNoVote dont seem to be used anywhere else either so i went with the prototype of making a method for a command,maybe as part of a general refactoring this could change

Reply:
I don't think the extra methods are necessarily bad; it can make things more readable since the functions are shorter.

Reply:
Agreeing with @AnnikaCodes I'd perfer the modularized approach here.